### DISCUSSION TITLE
Hello how are you doing?

### DISCUSSION THREAD

**Discussion by kasimgumel** [community/community#81304] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have a question pls on wordpress and formidable form pro, i i have created a form using formidable form plugin pls the form is for students i need the form to be when a students filled the form i will allow him to print all what he filled with serial number pls can i achieve this with custom code if someone having a code or shoe me the code pls i am waiting you?
Thanks

**Comment by queenofcorgis** [comment, marked as answer]
Hi @kasimgumel ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Wordpress and Formidable Form Pro. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by kasimgumel** [reply]
please can any one tell me  how to allow someone to search what he applied using formidable form pro plugin in wordpress
